connexion/decorators/security.py

```diff
@@ -17,14 +18,27 @@
 session.mount('https://', adapter)
 
 
+def get_tokeninfo_func(security_definition):
+    """
+    :type security_definition: dict
+    :rtype: function
+
+    >>> get_tokeninfo_url({'x-tokenInfoFunc': 'foo.bar>'})
+    '<function foo.bar>'
+    """
+    token_info_func = (security_definition.get("x-tokenInfoFunc") or
+                       os.environ.get('TOKENINFO_FUNC'))
```

Comment:
passing an (internal) function name via environment variable does not make sense I think

Reply:
This function name is the function in the users application, not in the library. So I think we should offer the same methods for the tokenInfoFunc as the tokenInfoUrl.

What is the reason you'd not want to include this?

Reply:
@MaicoTimmerman I can just not a find a good reason for it: the app and the swagger file should both be static (i.e. compile-time), environment variables are for runtime configuration and I don't see a good argument why you would want to switch the function at runtime (startup). We could leave it in, but I stumbled over it and it immediately raised the question "why? what's the benefit?".

Reply:
On the other hand I could imagine having a "noop" token validation function for local testing, so maybe your `TOKENINFO_FUNC` makes sense to overwrite it with empty value "" (no token validation).

Reply:
I'd personally prefer to keep it like it is currently, so can we proceed to merge the PR?